app/src/hid_listener.c

```diff
@@ -19,6 +19,22 @@ LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
 static int hid_listener_keycode_pressed(const struct zmk_keycode_state_changed *ev) {
     int err, explicit_mods_changed, implicit_mods_changed;
 
+    if (!(ev->usage_page == HID_USAGE_KEY && ev->keycode >= HID_USAGE_KEY_KEYBOARD_LEFTCONTROL &&
+          ev->keycode <= HID_USAGE_KEY_KEYBOARD_RIGHT_GUI) &&
```

Comment:
Check out `app/include/zmk/keys.h` for an `is_mod` function that should be used here (and our `hid.c` should probably be moved to using)

Reply:
Ah, this is what I was looking for, thanks! I just changed this instance since `hid.c` seems to be done with the usage page by the time it checks the mod, still, would be good to refactor those instances into their own is mod func. Out of scope for this PR though.